dev-workflow-ui/webContent/view/frame.xhtml

```diff
@@ -69,10 +69,17 @@
           }
 
           iframe.addEventListener("load", function() {
-            attachUnload();
-            // Just in case the change wasn't dispatched during the unload event...
-            dispatchChange();
-            iframe.style.visibility = 'visible';
+            try{
```

Comment:
please format the code to match other contents in here... seems like your curly braces have no leading space